### PR TITLE
show text if the cast list is empty in cast display component

### PR DESCRIPTION
**Pull request by aladdin91**
# Description
show text if the cast list is empty in cast display component

# Jira task link
https://hackyourfuture-dk.atlassian.net/browse/VIEW-79

# Screenshots (if applicable)

### CODE REVIEW

File: packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx [hunk 1]
```diff
@@ -81,49 +81,56 @@ export const TopCastDisplay = ({ movieID }) => {
...
+                    p: 1,
+                    textAlign: 'center',
+                    display: 'flex',
+                    flexDirection: 'column',
+                    justifyContent: 'space-between  ',
+                    alignItems: 'center',
+                  }}
+                >
+                  <StyledAvatar
+                    alt={`${castMember.full_name}`}
```
Comment (babak-f) [inline packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx:111]: You could put `castMember.full_name` directly here without the dollar sign and quote marks.

File: packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx [hunk 2]
```diff
@@ -81,49 +81,56 @@ export const TopCastDisplay = ({ movieID }) => {
...
+                    src={castMember.image_location}
+                    variant="circular"
+                  />
+                  <StyledCastTitle variant="caption">
+                    {`${castMember.full_name}`}
```
Comment (babak-f) [inline packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx:116]: Same as above

File: packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx [hunk 3]
```diff
@@ -32,14 +32,14 @@ export const TopCastDisplay = ({ movieID }) => {
...
   }, [movieID]);
-
   const handleShowFullCast = () => {
     setShowFullCast((previousFullCastState) => !previousFullCastState);
   };
 
   const StyledTypography = styled(Typography)`
     font-family: 'Inter';
     color: #ffffff;
+    font-size: 2rem;
```
Comment (babak-f) [inline packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx:42]: Using rem for font size, good choice! 👍

**Review by babak-f — APPROVED** [review]
Good job

**Review by shpomp — APPROVED** [review]

**Pull request merged**